pkg/util/resource_test.go

```diff
@@ -119,6 +141,9 @@ func TestGetPodEffectiveRequest(t *testing.T) {
 					},
 				})
 			}
+			if tt.podOverheadRequest != nil {
```

Comment:
ditto. We can assign it directly w/o the if check.

Reply:
done